### ISSUE TITLE
Using `LossScaleOptimizer` with TF 2.16.1

### ISSUE THREAD

**Issue by markub3327** [keras-team/keras#19479]
Hi,

   the tutorial for TF mixed precision is not working properly. If I try to use get_scaled_loss(), it returns an error about `LossScaleOptimizer` object has no attribute `get_scaled_loss`. If I look detailed to implementation of Keras 3.0, I can see only `scale_loss`.

Is it necessary to use it manually to rescale loss and the unscale gradients will be done automatically?

Please you can mention about it in docs about Keras 3.0, now I'm trying it like this:
```python
        with tf.GradientTape() as tape:
            y_pred = self(x, training=True)  # Forward pass
            # Compute the loss value
            loss = self.loss(y_true, y_pred)
            scaled_loss = self.optimizer.scale_loss(loss)

        # Compute gradients
        trainable_vars = self.network.trainable_variables
        gradients = tape.gradient(scaled_loss, trainable_vars)

        # Update weights
        self.optimizer.apply_gradients(zip(gradients, trainable_vars))
```

Thanks.

**Comment by fchollet** [comment]
> the tutorial for TF mixed precision

Can you provide a link?

If you are using `model.fit()` then loss scaling is automatically handled for you.

**Comment by markub3327** [comment]
@fchollet 

https://www.tensorflow.org/guide/mixed_precision
and there is nothing about `scale_loss` and using Mixed precision with a Custom training loop for TF!!!
https://keras.io/api/mixed_precision/policy/

Yes, I'm using `model.fit()`, but with a custom training loop. 
https://www.tensorflow.org/guide/keras/customizing_what_happens_in_fit#a_first_simple_example

**Comment by fchollet** [comment]
> Yes, I'm using model.fit(), but with a custom training loop.

You mean a custom `train_step`? In that case you should look at how `train_step` is implemented in the TF Trainer.
There's just this step after loss computation:

```python
if self.optimizer is not None:
    loss = self.optimizer.scale_loss(loss)
```

Basically, same as you were already using.
